# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1245904575 -7200
#      Thu Jun 25 06:36:15 2009 +0200
# Branch release-3-2-x
# Node ID 038885dcfb342c66cfcbd9204ee226a5d9e6bfe5
# Parent  a224089d84efa4f17321845fd147d402fc68ae26
remove ans from the list of former built-in variables

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2009-06-24  John W. Eaton  <jwe@octave.org>
+
+	* pt-assign.cc (former_built_in_variables): Remove "ans" from the list.
+
 2009-06-24  Jaroslav Hajek  <highegg@gmail.com>
 
 	* pt-eval.cc (do_unwind_protect_cleanup_code):
 	Workaround sig_atomic_t != int.
 
 2009-06-23  Jaroslav Hajek  <highegg@gmail.com>
 
 	* quit.h (octave_quit_exception): Delete.
diff --git a/src/pt-assign.cc b/src/pt-assign.cc
--- a/src/pt-assign.cc
+++ b/src/pt-assign.cc
@@ -59,17 +59,16 @@ static const char *former_built_in_varia
   "INFO_PROGRAM",
   "LOADPATH",
   "MAKEINFO_PROGRAM",
   "PAGER",
   "PS1",
   "PS2",
   "PS4",
   "__kluge_procbuf_delay__",
-  "ans",
   "automatic_replot",
   "beep_on_error",
   "completion_append_char",
   "crash_dumps_octave_core",
   "current_script_file_name",
   "debug_on_error",
   "debug_on_interrupt",
   "debug_on_warning",
